zebra-state/src/request.rs

```diff
@@ -224,6 +224,7 @@ pub struct ContextuallyVerifiedBlock {
 }
 
 /// Wraps note commitment trees and the history tree together.
+#[derive(Default)]
```

Comment:
We usually derive a bunch of standard traits, not sure why that didn't happen here.

Let's also document that the default represents the genesis treestate.

```suggestion
#[derive(Clone, Debug, Default, Eq, PartialEq)]
```